E/extrae/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env julia
+using BinaryBuilder, Pkg
+
+name = "extrae"
+version = v"4.0.1"
+sources = [
+    ArchiveSource(
+        "https://github.com/bsc-performance-tools/extrae/archive/refs/tags/$(version).tar.gz",
+        "e6765eb087be3f3c162e08d65f425de5f26912811392527d56ecd75d4fb6b99d"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/extrae-*
+autoreconf -fvi
+./configure \
```

Comment:
https://docs.binarybuilder.org/stable/build_tips/#Autoconfigure-builds
```suggestion
./configure \
    --build=${MACHTYPE} \
    --host=${target} \
```

Reply:
Ah nice thanks. I'll retry later today

Reply:
This is still necessary

Reply:
ah sorry no idea why I dropped it. :confounded:

Reply:
ah yes, I dropped it because the configure script can't manage that:
```
[10:01:33] configure: error: in `/workspace/srcdir/extrae-4.0.1':
[10:01:33] configure: error: cannot run test program while cross compiling
[10:01:33] See `config.log' for more details
```
...even without actually cross-compiling.

What is the best solution? Not sure if patching it is viable here; I'll leave a note to the issue in extrae there and update the PR once they fix it?